src/main/java/br/edu/ufcg/computacao/alumni/core/holders/AlumniHolder.java

```diff
@@ -137,6 +143,34 @@ private String getToken() throws EurecaException {
         return page;
     }
 
+    private synchronized Collection<UfcgAlumnusData> getAlumniFilteredData(String admission, String graduation) {
+        Set<UfcgAlumnusData> alumni = new HashSet<>();
+        alumni.addAll(filterAlumniBy("admission", admission));
+        alumni.addAll(filterAlumniBy("graduation", graduation));
```

Comment:
Qual o comportamento de addAll se graduation ou admission for null?

Reply:
Caso o segundo parâmetro do método filterAlumniBy seja null, no for desse método o switch cai no default e não adiciona. Isto é, caso essa função getAlumniFilteredData for chamada, sabemos que pelo menos um (admission ou gradiation) não é null, então será considerado apenas admission, apenas graduation ou os dois atributos.